File: year2020/month11/day1118/ms_1118.cs
using System;

namespace algorithm_103
{
    class ms_1118
    {
        static void Main(string[] args)
        {
            String[] data = { "flower", "rose", "lily", "daffodil", "azalea" };
            char input = Convert.ToChar(Console.ReadLine());
            int count = 0;

            for(int i = 0; i < data.Length; i++)
            {
                char second = Convert.ToChar(data[i][1]);
                char third = Convert.ToChar(data[i][2]);

                if((second == input) || (third == input)) 
                {
                    Console.WriteLine(data[i]);
                    count++;
                }
            }

            Console.WriteLine(count);
        }
    }
}

File: year2020/month11/day1118/ms_1118.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string data[5] = { "flower", "rose", "lily", "daffodil", "aza;ea" };
	char input;
	int count = 0;

	cin >> input;

	for (int i = 0; i < 5; i++) {
		char second = data[i][1];
		char third = data[i][2];

		if ((second == input) || (third == input)) {
			cout << data[i] << endl;
			count++;
		}
	}

	cout << count << endl;

	return 0;
}